src/static/major.json

```diff
@@ -420,12 +438,6 @@
       "chineseName": "性別研究（兩年制課程）",
       "faculties": ["SSF"]
     },
-    {
-      "code": "GLBSN",
-      "englishName": "Global Studies (2-year programme)",
-      "chineseName": "全球研究（兩年制課程）",
-      "faculties": ["SSF"]
-    },
```

Comment:
Should we keep old programs for backward compatibility?
And also add "inactive: true" to them?

Reply:
Sure, and let me think think whether to use boolean or time
PS: AFAIK GLBSN merged into senior-entry GLSDN some years ago, maybe that's why the code is deleted from RES?

Reply:
Or let me check if this code is used right now